bindgen/options/mod.rs

```diff
@@ -1416,6 +1416,21 @@ options! {
         },
         as_args: "--generate-block",
     },
+    /// Whether to generate strings as `CStr`.
+    generate_cstr: bool {
+        methods: {
+            /// Set whether strings should be generated as `&CStr` instead of `&[u8]`.
+            ///
+            /// This option is disabled by default but will become enabled by default in a future
```

Comment:
Could you make clearer that this feature won't have any effect if the rust target version is lower than `1.59`?